src/index.html

```diff
@@ -4,9 +4,470 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <container class="container-for-header">
+      <header class="header container-for-header__position">
+
+        <img
+          src="images/logo.png"
+          alt="Лого интернет-магазина Mojo"
+          class="image"
+        >
+
+        <nav class="nav">
+          <ul class="nav__list">
+
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">
+                APPLE
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                SAMSUNG
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Смартфоны и телефоны
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Гаджеты
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Планшеты
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Фото
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Видео
+              </a>
+            </li>
+
+          </ul>
+        </nav>
+      </header>
+    </container>
+
+    <main class="main">
+      <content class="content-home-page">
```

Comment:
```suggestion
      <div class="content-home-page">
```

Let's use 'div'. 'Content' is deprecated